terraform/validators.tf

```diff
@@ -202,44 +202,57 @@ locals {
   validator_command      = var.log_to_file || var.enable_logstash ? jsonencode(["bash", "-c", "/docker-run-dynamic.sh >> ${var.log_path} 2>&1"]) : ""
   aws_elasticsearch_host = var.enable_logstash ? join(",", aws_elasticsearch_domain.logging.*.endpoint) : ""
   logstash_config        = "input { file { path => '${var.structlog_path}'\\n codec => 'json'\\n}}\\n filter {  json {  \\nsource => 'message'\\n}}\\n output {  amazon_es { \\nhosts => ['https://${local.aws_elasticsearch_host}']\\nregion => 'us-west-2'\\nindex => 'validator-logs-%%{+YYYY.MM.dd}'\\n}}"
+  chain_id               = "tf-testnet-${terraform.workspace}"
```

Comment:
I'm assuming this only applies to testnet?  I don't really have a lot of context on how the terraform configurations are set up

Reply:
Anything deployed with these Terraform configs. The persistent ones are prod and devnet.